0.59.md

```diff
@@ -0,0 +1,348 @@
+# 0.59
+
+Welcome to the February 2019 release of React Native! This is another big release with 644 commits by 88 contributors. Contributions also come in other forms, so *thank you* for maintaining issues, fostering communities, and teaching people about React Native. This month brings a number of highly anticipated changes, and we hope you enjoy them.
+
+## Hooks are here
+
+React Hooks are here, which let you reuse stateful logic across components. This addition has a lot of buzz about it; take a look at some of the wonderful resources below to learn more:
+
+> - [Introducing Hooks](https://reactjs.org/docs/hooks-intro.html) explains why we’re adding Hooks to React.
+> - [Hooks at a Glance](https://reactjs.org/docs/hooks-overview.html) is a fast-paced overview of the built-in Hooks.
+> - [Building Your Own Hooks](https://reactjs.org/docs/hooks-custom.html) demonstrates code reuse with custom Hooks.
+> - [Making Sense of React Hooks](https://medium.com/@dan_abramov/making-sense-of-react-hooks-fdbde8803889) explores the new possibilities unlocked by Hooks.
+> - [useHooks.com](https://usehooks.com/) showcases community-maintained Hooks recipes and demos.
+
+Be sure to give this a try in your apps. We hope that you find the reuse as exciting as we do.
+
+## Big Performance Gains and 64-bit Support on Android
+
+The Android platform on React Native has an embedded WebKit JavaScript engine that powers most of your application (it's called the "JavaScript Core" or "JSC"). The JSC was quite out of date, which meant that a lot of modern JavaScript features weren't supported. Even worse, it performed poorly compared iOS's modern JSC. With this release, that all changes.
+
+Thanks to some awesome work by [@DanielZlotin](https://twitter.com/danielzlotin), the JSC has caught up with the past few years. This brings with it 64-bit support, modern JavaScript support, and [big performance improvements](https://github.com/react-native-community/jsc-android-buildscripts/tree/master/measure). Kudos to Daniel for also making this a maintainable process now so that we can take advantage of future WebKit improvements without so much legwork.
+
+## Faster app launches with inline requires
+
+We want to help people have performant React Native apps by default and are working to bring Facebook's optimizations to the community. [@eli_white](https://twitter.com/eli_white) did incredible work to help applications load resources as needed rather than slowing down launch. This feature is called "inline requires", as it lets the packager identify components to be lazy loaded. Apps with a deep and varied component architecture will see the most improvement.
```

Comment:
I defffffinitely didn’t do this work. This optimization has been around for years in our performance docs but people didn’t know it existed. We are working to turn it on by default to help people fall into the pit of success.

Reply:
(also, since we are in GitHub land I think it would be better to use GitHub profiles, as a general rule)